backvendor/workingtree.go

```diff
@@ -68,21 +68,21 @@ func (wt *WorkingTree) SemVerTags() ([]string, error) {
 		return nil, err
 	}
 	semvers := make(semver.Collection, 0)
-	semvertags := make(map[*semver.Version]string)
+	semVerTags := make(map[*semver.Version]string)
```

Comment:
Wonder whether this one should be `semverTags`, for consistency with `semvers` (which is all lowercase due to the `semver` import I guess?).

Reply:
Good question. The top two hits on godoc.org for semvar use _version_ as the base of identifiers, not _semver_, so that bright idea doesn't help.
https://godoc.org/github.com/coreos/go-semver/semver
https://godoc.org/github.com/blang/semver
But then I see https://godoc.org/github.com/Masterminds/semver that's being used also does that so perhaps it's backvendor being influenced by the package name that's wrong.

Reply:
Rebased with _semver_ treated as a single word. Switch to _version_ root would be a separate PR.